google/resource_logging_project_sink.go

```diff
@@ -96,3 +104,15 @@ func resourceLoggingProjectSinkDelete(d *schema.ResourceData, meta interface{})
 	d.SetId("")
 	return nil
 }
+
+func resourceLoggingProjectSinkImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	parts := strings.Split(d.Id(), "/")
+	if len(parts) != 4 {
```

Comment:
Can you add a regex check and a default project check here? Similar to [KMS key ring](https://github.com/terraform-providers/terraform-provider-google/blob/cc5f77e32e927c3df81fab6abc4ee791c39bab5c/google/resource_kms_key_ring.go#L133-L160)

Reply:
Done. Called `parseLoggingSinkId`.